### PR TITLE
add longitudinal maneuvers

### PR DESCRIPTION
**Pull request by sshane**
from opendbc: https://github.com/commaai/opendbc/blob/ef7102a8ae2334d77ff4b0be512c073e81b01d18/examples/longitudinal-profiles.py

- [x] uses alert manager similarly to joystick alert
- [x] enable by setting `LongitudinalManeuverMode`
- [x] give remote/local route to generate_report.py
  - [x] checks for user override or not fully enabled
  - [ ] relative time
  - [ ] checks for pitch
  - [ ] localizer accel (not in this PR, there's too many accel signals on the plot and we should make sure to keep the report simple)

### CODE REVIEW

File: tools/longitudinal_maneuvers/maneuversd.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  maneuver = None
+
+  while True:
+    sm.update()
+
+    if maneuver is None:
+      maneuver = next(maneuvers, None)
+
+    if maneuver is None:
+      print('We are done!')
```
Comment (adeebshihadeh) [inline tools/longitudinal_maneuvers/maneuversd.py:128]: I think wrapping around is better than no maneuvers

File: tools/longitudinal_maneuvers/maneuversd.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+from dataclasses import dataclass
+
+from cereal import messaging, car
+from opendbc.car.common.conversions import Conversions
```
Comment (adeebshihadeh) [inline tools/longitudinal_maneuvers/maneuversd.py:5]: we should just rename Conversions to something short. `Cnvrt`?

File: tools/longitudinal_maneuvers/maneuversd.py (hunk 1 above)
Comment (sshane) [reply tools/longitudinal_maneuvers/maneuversd.py:128]: Without any alert it may be easy to miss that the report is done

**Comment by sshane** [comment]
@adeebshihadeh Anything else you think the first iteration needs?

**Pull request merged**